src/main/kotlin/io/snyk/plugin/analytics/EventPropertiesProvider.kt

```diff
@@ -0,0 +1,60 @@
+package io.snyk.plugin.analytics
+
+import ai.deepcode.javaclient.core.SuggestionForFile
+import io.snyk.plugin.cli.CliResult
+import io.snyk.plugin.cli.Vulnerability
+import io.snyk.plugin.services.SnykApplicationSettingsStateService
+import io.snyk.plugin.snykcode.SnykCodeResults
+import io.snyk.plugin.snykcode.severityAsString
+
+object EventPropertiesProvider {
```

Comment:
Just a suggestion: maybe use just functions? Like we have here  [Utils.kt](https://github.com/snyk/snyk-intellij-plugin/blob/master/src/main/kotlin/io/snyk/plugin/Utils.kt)

Reply:
Discussed together about context. We leave it for now until analytics will be extracted into a separate library.
Thanks @aldanchenko